Documentation/Helm-Charts/operator-chart.md

```diff
@@ -74,6 +74,7 @@ The following tables lists the configurable parameters of the rook-operator char
 | `csi.cephFSFSGroupPolicy`           | Policy for modifying a volume's ownership or permissions when the CephFS PVC is being mounted                                                    | `ReadWriteOnceWithFSType`                                 |
 | `csi.nfsFSGroupPolicy`              | Policy for modifying a volume's ownership or permissions when the NFS PVC is being mounted                                                       | `ReadWriteOnceWithFSType`                                 |
 | `csi.logLevel`                      | Set logging level for csi containers. Supported values from 0 to 5. 0 for general useful logs, 5 for trace level verbosity.                      | `0`                                                       |
```

Comment:
update comment for `csi.logLevel` to `cephCSI containers maintained by the cephCSI` to reflect it only applicable for cephcsi containers

Reply:
done